plugins/UFM_NDT_Plugin/build/config/ndt_logrotate.conf

```diff
@@ -0,0 +1,9 @@
+/log/ndt.log {
+    compress
+    rotate 10
```

Comment:
rotate and size should be configurable in the NDT plugin configuration.

Reply:
? how? it is external mechanism not related to plugin implementation.